### DISCUSSION TITLE
HasManyThrough with Many to Many

### DISCUSSION THREAD

**Discussion by kush-prof** [adonisjs/core#2688] [Help]
Hi, is it possible to have a has many through relationship  between `A` `B` `C`, where `B` and `C` have many to many and not one to many.

IMO the concept of HasManyThrough applies here as well. What do you think?

**Comment by thetutlage** [comment]
Hard to provide an answer with an abstract question. Can you share a real practical example?

**Reply by kush-prof** [reply]
Hi sorry for the late reply.

User (parents) have children and children have classes, so a `parent` also has many classes assigned to his account.

Now if I want to fetch all the classes of a parent can I use `HasManyThrough` relationship?
